sklearn/tests/test_multioutput.py

```diff
@@ -864,3 +864,19 @@ def test_multioutput_regressor_has_partial_fit():
     msg = "This 'MultiOutputRegressor' has no attribute 'partial_fit'"
     with pytest.raises(AttributeError, match=msg):
         getattr(est, "partial_fit")
+
+
+# TODO(1.8):  remove when deprecated `base_estimator` is removed
+@pytest.mark.parametrize("Estimator", [ClassifierChain, RegressorChain])
+def test_base_estimator_deprecation(Estimator):
+    # Check that deprecation warning is raised when
+    # calling .fit()
```

Comment:
```suggestion
    # Check that FutureWarning is raised when calling .fit()
```

Maybe change it to FutureWarning, because this is what we test for.